libs/shared/services/src/lib/packs.service.ts

```diff
@@ -1334,7 +1337,7 @@ export default class PacksService {
               variables: {
                 amount: `${formatIntToFloat(
                   selectedBid.amount,
-                  Configuration.currency // TODO: receive as argument
+                  this.currency // TODO: receive as argument
```

Comment:
TODO comment can be removed, I think?

```suggestion
                  this.currency
```